adhocteam/Head-Start-TTADP
# Alert on draft save

**Description of change**
Alert is shown after a draft is saved

Also fixed two bugs related to page state

1) Pages would set their state to the page they switch to. So if I browsed to a `Not Started` page from a `Completed` page that `Not Started` would be set to `Completed` after browsing away, even if the page was not completed.
2) Switching between grantee/non-grantee didn't properly handle the state of the activity summary page

**How to test**

1) Pull down changes
2) Save draft reports to make sure the "save draft" dialog works
3) Save and browse pages to make sure page state is updated correctly

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/375

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
